### PR TITLE
Add some routing to frontend example

### PR DESCRIPTION
**Pull request by 72636c**

### CODE REVIEW

File: fe/example/src/App/App.treat.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { globalStyle } from 'sku/treat';
+
+globalStyle('body', {
+  height: '100vh',
+});
+
+globalStyle('#app, #app > div, #app > div > div', {
```
Comment (aaronvanston) [inline fe/example/src/App/App.treat.ts:7]: This structure should not change, so I think it's fine. However, these nested styles could hurt us down the track. There should be ways to isolate/ attach the styles to the correct divs to push the height to 100vh.

Comment (72636c) [reply fe/example/src/App/App.treat.ts:7]: Yeah this is gross, I'll have a look

Comment (72636c) [reply fe/example/src/App/App.treat.ts]: That works a treat! Ryan always gets salty about 100vh breaking his mobile Safari though

Comment (etaoins) [reply fe/example/src/App/App.treat.ts]: Mobile Safari is living in a hell it created for itself. Let it burn.

**Pull request merged**